src/raven/client.clj

```diff
@@ -120,17 +119,30 @@
         param  (fn [[k v]] (format "sentry_%s=%s" (name k) v))]
     (str "Sentry " (str/join ", " (map param params)))))
 
+(defn merged-payload
+  "Return a payload map depending on the type of the event."
+  [ev ts pid uuid]
+  (merge (default-payload ts)
+         (cond
+           (map? ev)       ev
+           (exception? ev) (exception->ev ev)
+           :else           {:message (str ev)})
+         {:event_id uuid
+          :project  pid}))
+
+(defn validate-payload
```

Comment:
I would use an `assert` here to throw on malformed payloads.

Reply:
Done.